src/docs/config/overview.md

```diff
@@ -166,6 +167,30 @@ The `preamble` configuration is a `string` that represents a preamble in the mai
 preamble: 'Built with Stencil'
 ```
 
+## sourceMap
+
+*default: `false`*
+
+When set to `true`, sourcemaps will be generated for a project.
+
+```tsx
+sourceMap: true
+```
+
+Sourcemaps create a translation between Stencil components that are written in TypeScript & JSX and JavaScript that is
+output by Stencil. They allow for a better debugging experience for Stencil components, in that they allow external
+tools (such as an Integrated Development Environment) to add breakpoints directly in the original source code. This
```

Comment:
```suggestion
tools (such as an Integrated Development Environment) to add breakpoints directly in the original source code, which
```

Reply:
I think this suggestion results in too much of a run-on sentence

Reply:
When used with both above suggestions, the concepts should adequately break themselves up and improve readability.